XenAdmin/Wizards/NewPolicyWizard/NewPolicySnapshotTypePage.resx

```diff
@@ -227,7 +227,7 @@
 
 1. The VM is running or suspended at the time the snapshot is scheduled.
 2. The VM has [Citrix VM Tools] installed.
-3. The VM does NOT have dedicated GPU assigned to it.</value>
+3. The VM does NOT have dedicated Intel/AMD GPU assigned to it(Nvidia GPU is supported).</value>

```

Comment:
This is only true for Naples or greater hosts, so a version check will be needed here. 
Also I suggest that we don't specify the vendor names here, maybe we can change the message to something neutral, like: The VM does NOT have dedicated GPU assigned that does not support disk and memory snapshots.

Reply:
General point: there should be a space between a word and the following parenthesis, i.e. instead of `to it(` it should be `to it (`.
Alternative message: `3. If the VM has a dedicated GPU assigned to it, this GPU supports disk and memory snapshots.`

Reply:
Thank you all for the comprehensive comments.

I think the neutral way is good, like @kc284 wrote: `3. If the VM has a dedicated GPU assigned to it, this GPU supports disk and memory snapshots.`  This is nice.